web/ASC.Web.Common/src/components/routing/externalRedirect.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { PageLayout, Loader } from "asc-web-components";
+
+export class ExternalRedirect extends Component {
+  constructor(props) {
+    super();
+    this.state = {
+      location: props.to
+    };
+  }
+
+  componentDidMount() {
+    window.location.replace(this.state.location);
```

Comment:
If you pass props as an attribute to the super() you can fetch target location from this.props.to and remove state variable.